pom.xml

```diff
@@ -153,7 +153,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.build.outputTimestamp>${git.commit.time}</project.build.outputTimestamp>
     <slf4j.version>1.7.30</slf4j.version>
-    <jackson.version>2.13.3</jackson.version>
+    <jackson.version>2.13.5</jackson.version>
```

Comment:
this change looks acceptable. but i would prefer if each version change is self contained so we may merge and evaluate any issues separately, rather than merging several changes all at once.

Reply:
see https://github.com/Alluxio/alluxio/pull/17000